server/server.go

```diff
@@ -475,7 +476,7 @@ func (s *Server) Run(ctx context.Context) error {
 	httpServer := &http.Server{
 		Addr: fmt.Sprintf(":%d", s.config.HTTPServer.Addr),
 		Handler: cors.New(cors.Options{
-			AllowedOrigins:   []string{"*"},
+			AllowedOrigins:   s.config.HTTPServer.CORSAllowedOrigins,
```

Comment:
Do you think we should also allow the other options to be configurable as well?

Reply:
`AllowedHeaders` should be configurable.  As for `AllowCredentials` and `AllowedMethods`, we probably don't want to change as they are related to our authentication + provided methods.